packages/core/saved-objects/core-saved-objects-api-server-internal/src/test_helpers/repository.test.common.ts

```diff
@@ -602,6 +612,7 @@ export const getMockBulkCreateResponse = (
             references,
             ...mockTimestampFieldsWithCreated,
             typeMigrationVersion: typeMigrationVersion || migrationVersion?.[type] || '1.1.1',
+            managed: managed || docManaged || false,
```

Comment:
NIT: even if it doesn't have impact on the logic, `managed ?? docManaged ?? false` seems more accurate?